.github/workflows/tg-unit-tests-impl.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
           echo "TT_METAL_HOME=$(pwd)" >> $GITHUB_ENV
       - name: Build UMD device and tests
         run: |
-          cmake -B build -G Ninja
+          cmake -B build -G Ninja -DTT_UMD_BUILD_TESTS=ON
```

Comment:
Do we run UMD tests in TG?

Reply:
The existing build command was explicitly building `umd_tests` so I aimed to keep things equivalent.
So it appears that way to me?

Reply:
Answer: Yes we do: https://github.com/tenstorrent/tt-metal/actions/runs/11130875227/job/30931931484

If you want to remove that we can do it in a future PR.